arch/arm/boot/dts/overlays/dt-blob-overlay.dts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Device tree blob overlay for Revolution Pi by KUNBUS
+ *
+ * RevPi Devices
+ */
+
+#include "rpi-dt-blob-overlay.dts"
+
+
+/ {
+	videocore {
+		pins_cm {
+			pin_config {
+				/* For Core SPI0 CLK */
+				pin@p39 { function = "spi"; termination = "no_pulling"; drive_strength_mA = < 8 >; };
+			}; // pin_config
+			pin_defines {
+			}; // pin_defines
+		};
+
+		pins_cm3 {
+			pin_config {
+				/* For Core 3 and Connect SPI0 CLK */
```

Comment:
as this? "/* For Core S and Connect S set the drive strength for the whole GPIO bank */

Reply:
My suggestion would be:

```
Increase drive strength in order to improve the signal quality (eg. SPI CLK on Core and Connect)
NOTE: This will increase the drive strength for the whole GPIO bank 1 and not only pin 39
```

Reply:
good idea, thanks

Reply:
but I am not quite clear is, why it is configured on pin 39, not other pin, or all pins ?

Reply:
From https://github.com/raspberrypi/documentation/blob/develop/documentation/asciidoc/computers/configuration/pin-configuration.adoc: 
```
drive_strength_mA The drive strength is used to set a strength for the pins. Please note that you can only specify a single drive strength for the bank. <8> and <16> are valid values.
```

So, if you set the value for one pin all others will have the same value.